tests/Unit/Integrations/IntegrationsLoaderTest.php

```diff
@@ -175,6 +175,7 @@ public function testWeDidNotForgetToRegisterALibraryForAutoLoading()
             $excluded[] = 'symfony';
             $excluded[] = 'wordpress';
             $excluded[] = 'yii';
+            $excluded[] = 'roadrunner';
```

Comment:
This list appears to be in alphabetical order; can you insert it in the right place?